scripts/ci/release.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Copyright 2017 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dir=$(dirname $0)
+projectRoot=${dir}/../..
+
+if [ -z "$DOCKER_NAMESPACE" ]; then
+  DOCKER_NAMESPACE="gcr.io/$(gcloud info \
+                | awk '/^Project: / { print $2 }' \
+                | sed 's/\[//'  \
+                | sed 's/\]//')"
+fi
+
+${projectRoot}/scripts/build.sh \
+  -Ddocker.project.namespace=${DOCKER_NAMESPACE}
```

Comment:
This feels strange to be passing in a java system property to a bash script like this. Can you change it so that the ${DOCKER_NAMESPACE} is used as the argument? Then in the `build.sh` script, you can add the maven property name.